### ISSUE TITLE
bug: DGR error message, when using numbers as elements

### ISSUE THREAD

**Issue by oliwial23** [morazanm/fsm#64]
## Describe the bug
DGR graphing does not work, when using numbers as elements. 

## Steps to Reproduce
1) Run the code
```racket
(define PROP-BINARY (make-dfa '(S N)
                              '(0 1)
                              'S
                              '(N)
                              '((S 1 N)
                                (N 0 N)
                                (N 1 N))))

(sm-visualize PROP-BINARY)
```
2) Click on the DGR button

## Expected Behavior
A graph (.dot img) should appear. Instead, there is an error message.

## OS Observed On
macOS Ventura 13.1

## Screenshots
<img width="560" alt="Bildschirm­foto 2023-02-23 um 13 02 00" src="https://user-images.githubusercontent.com/119520637/220991586-905f4bac-bec5-4da7-812d-adf09b0cf51b.png">

**Comment by jschappel** [comment]
@morazanm Should this machine even be building? The grammar for [make-dfa](https://morazanm.github.io/fsm/fsm/State_Machine_Constructors.html) says that the second argument should be the alphabet. According to the docs the [alphabet](https://morazanm.github.io/fsm/fsm/Data_Definitions.html) is " `A list of lowercase symbols with a string representation of length one not including EMP`". A number by its self in racket is never a symbol
```racket
> (symbol? '0)
#f
> (symbol? '0a)
#t
```

**Comment by jschappel** [comment]
I spoke with @morazanm and confirmed that numbers should be allowed as a alpha. This was fixed by #65
